cmd/kyma/alpha/create/module/module.go

```diff
@@ -320,7 +323,7 @@ func (cmd *command) Run(ctx context.Context) error {
 	// Security Scan
 	if cmd.opts.SecurityScanConfig != "" && gitPath != "" { // security scan is only supported for target git repositories
 		cmd.NewStep("Configuring security scanning...")
-		if _, err = osFS.Stat(cmd.opts.SecurityScanConfig); err == nil {
+		if err = files.IsFileExists(cmd.opts.SecurityScanConfig); err == nil {
```

Comment:
This function I believe can't handle relative path, say if I provide `../btp-manager/sec-scanners-config.yaml`, it complains file not exists